liiatools_pipeline/ops/common_org.py

```diff
@@ -59,13 +63,16 @@ def create_org_session_folder() -> FS:
 def create_reports(
     session_folder: FS,
 ):
-    export_folder = workspace_folder().makedirs("current/ssda903", recreate=True)
+    export_folder = workspace_folder().makedirs(f"current/{dataset()}", recreate=True)
     aggregate = DataframeAggregator(session_folder, pipeline_config())
     aggregate_data = aggregate.current()
 
     for report in ["PAN", "SUFFICIENCY"]:
         report_folder = export_folder.makedirs(report, recreate=True)
         report_data = prepare_export(aggregate_data, pipeline_config(), profile=report)
+        report_data.data.export(report_folder, f"{dataset()}", "csv")
+        report_data.data.export(shared_folder(), f"{report}_{dataset()}_", "csv")
+
```

Comment:
move ssda903 specific code (dim_tables onwards, I can't make comments on that line though) to ssda903_org.py